test/browser/browser.sh

```diff
@@ -29,6 +29,12 @@ if grep -q platform:el8 /etc/os-release; then
     dnf module switch-to -y nodejs:16
 fi
 
+# HACK: nodejs 20 is very segfaulty: https://bugzilla.redhat.com/show_bug.cgi?id=2190075
+if grep -q VERSION_ID=39 /etc/os-release; then
+    dnf install -y nodejs18
```

Comment:
dnf module switch-to does not work?

Reply:
This is Fedora, so no modules. The nodejs:16 above is for RHEL 8.